### PR TITLE
[i18n sprint] moving translations for search help tips from ftl files to property files

### PR DESCRIPTION
**Pull request by chenejac**
[Companion PR](https://github.com/vivo-project/Vitro/pull/333)
# What does this pull request do? #
Moves translations from ftl pages to properties.
# How to test # 
Page to test /searchHelp
# Translations review # 
The sixth advanced tip is missing for Germany, at the moment it is in English
Translations for other languages didn't change, just copied from ftl files to property files.

### CODE REVIEW

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 1]
```diff
@@ -530,6 +530,19 @@ search_for = Suche nach '' {0} ''
...
+advanced_search_tips_header = Tipps für Fortgeschrittene
+advanced_search_tip_one = Wenn Sie mehr als einen Begriff angeben, wird die Suche nur Ergebnisse liefern, die alle Begriffe enthalten, es sei denn Sie ergänzen "OR" - z.B. <i>Huhn</i> OR <i>Ei</i>.
+advanced_search_tip_two = "NOT" kann zur Einschränkung der Suche verwendet werden, - z.B. <i>Klima</i> NOT <i>Veränderung</i>.
+advanced_search_tip_three = Wortgruppensuchen können mit Booleschen Operatoren kombiniert werden - z.B. "<i>Klimaveränderung</i>" OR "<i>Globale Erwärmung</i>".
+advanced_search_tip_four = Wortvariationen werden auch gefunden - Bsp.: <i>Sequenz</i> findet <i>Sequenzen</i> and <i>Sequenzierung</i>.
+advanced_search_tip_five = Verwenden Sie das Wildcard-Zeichen * um mehr Variationen zu finden - Bsp.: <i>nano*</i> findet <i>Nanotechnology</i> sowie <i>Nanofabrikation</i>.
+advanced_search_tip_six = Search uses shortened versions of words -- e.g., a search for <i>cogniti*</i> finds nothing, while <i>cognit*</i> finds both <i>cognitive</i> and <i>cognition</i>.
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
advanced_search_tip_six = Die Suche verwendet verkürzte Versionen von Wörtern – z. B. findet eine Suche nach <i>cogniti*</i> nichts, während <i>cognit*</i> sowohl <i>cognitive</i> als auch <i>cognition</i> findet.
```

**Review by bkampe — CHANGES_REQUESTED** [review]

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 2]
```diff
@@ -530,6 +530,19 @@ search_for = Suche nach '' {0} ''
...
+advanced_search_tips_header = Tipps für Fortgeschrittene
+advanced_search_tip_one = Wenn Sie mehr als einen Begriff angeben, wird die Suche nur Ergebnisse liefern, die alle Begriffe enthalten, es sei denn Sie ergänzen "OR" - z.B. <i>Huhn</i> OR <i>Ei</i>.
+advanced_search_tip_two = "NOT" kann zur Einschränkung der Suche verwendet werden, - z.B. <i>Klima</i> NOT <i>Veränderung</i>.
+advanced_search_tip_three = Wortgruppensuchen können mit Booleschen Operatoren kombiniert werden - z.B. "<i>Klimaveränderung</i>" OR "<i>Globale Erwärmung</i>".
+advanced_search_tip_four = Wortvariationen werden auch gefunden - Bsp.: <i>Sequenz</i> findet <i>Sequenzen</i> and <i>Sequenzierung</i>.
+advanced_search_tip_five = Verwenden Sie das Wildcard-Zeichen * um mehr Variationen zu finden - Bsp.: <i>nano*</i> findet <i>Nanotechnology</i> sowie <i>Nanofabrikation</i>.
+advanced_search_tip_six = Die Suche verwendet verkürzte Versionen von Wörtern – z. B. findet eine Suche nach <i>cogniti*</i> nichts, während <i>cognit*</i> sowohl <i>cognitive</i> als auch <i>cognition findet </i>.
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
advanced_search_tip_six = Die Suche verwendet verkürzte Versionen von Wörtern – z. B. findet eine Suche nach <i>cogniti*</i> nichts, während <i>cognit*</i> sowohl <i>cognitive</i> als auch <i>cognition</i> findet.
```

**Review by bkampe — CHANGES_REQUESTED** [review]

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 3]
```diff
@@ -836,6 +849,7 @@ or_enter_valid_address = oder geben Sie eine andere vollständige und gültige E
 share_profile_uri = Die URI für dieses Profil teilen
 view_profile_in_rdf = Profil in RDF-Format anzeigen
 close = schließen
+close_capitalized = Schließen
 
 error_processing_labels = Fehler bei der Verarbeitung der Anfrage: Die deaktivierten Labels konnte nicht gelöscht werden.
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
error_processing_labels = Fehler bei der Verarbeitung der Anfrage: Die deaktivierten Labels konnten nicht gelöscht werden.
```

**Review by bkampe — CHANGES_REQUESTED** [review]
Minor spelling correction

File: ru_RU/webapp/src/main/webapp/i18n/all_ru_RU.properties [hunk 4]
```diff
@@ -541,6 +541,19 @@ search_for = Результаты поиска по запросу «{0}»
 search_results_for = Результаты поиска по запросу
 limited_to_type = \u0020ограничены по типу объектов
 search_help = Помощь по поиску
+search_tips_header = Советы для поиска
+back_to_results = Back to results
```
Comment (litvinovg) [inline ru_RU/webapp/src/main/webapp/i18n/all_ru_RU.properties]:
```suggestion
back_to_results = Назад к результатам поиска
```

**Review by litvinovg — APPROVED** [review]
Looks good. Take a look at my suggestion.

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 5]
```diff
@@ -530,6 +530,19 @@ search_for = Suche nach '' {0} ''
 search_results_for = Suchergebnisse für
 limited_to_type = beschränkt auf den Typ
 search_help = Suchhilfe
+search_tips_header = Suchtipps
+back_to_results = Zurück zu den Suchergebnissen
+search_tip_one = Halten Sie es einfach! Verwenden Sie kurze, einzelne Begriffe, wenn ihre Suche zu viele Ergebnisse liefert.
+search_tip_two = Verwenden Sie Anführungszeichen für die Suche nach eine Phrase - z.B. "<i>protein folding</i>".
+search_tip_three = Mit Ausnahme Boolescher Operatoren, sind Suchen <strong>nicht</strong> case-sensitive, also sind "Genf" und "genf" identisch.
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
search_tip_three = Mit Ausnahme boolescher Operatoren, sind Suchen <strong>nicht</strong> case-sensitive, also sind "Genf" und "genf" identisch.
```

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 6]
```diff
@@ -530,6 +530,19 @@ search_for = Suche nach '' {0} ''
...
+search_tip_four = Wenn Sie sich unsicher über die korrekte Rechtschreibung sind, nutzen Sie ~ am Ende Ihres Suchbegriffes -- Bsp.: <i>cabage~</i> findet <i>cabbage</i>, <i>steven~</i> findet <i>Stephen</i> und <i>Stefan</i> (und andere, ähnliche Namen).
+advanced_search_tips_header = Tipps für Fortgeschrittene
+advanced_search_tip_one = Wenn Sie mehr als einen Begriff angeben, wird die Suche nur Ergebnisse liefern, die alle Begriffe enthalten, es sei denn Sie ergänzen "OR" - z.B. <i>Huhn</i> OR <i>Ei</i>.
+advanced_search_tip_two = "NOT" kann zur Einschränkung der Suche verwendet werden, - z.B. <i>Klima</i> NOT <i>Veränderung</i>.
+advanced_search_tip_three = Wortgruppensuchen können mit Booleschen Operatoren kombiniert werden - z.B. "<i>Klimaveränderung</i>" OR "<i>Globale Erwärmung</i>".
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
advanced_search_tip_three = Wortgruppensuchen können mit booleschen Operatoren kombiniert werden - z.B. "<i>Klimaveränderung</i>" OR "<i>Globale Erwärmung</i>".
```

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 7]
```diff
@@ -530,6 +530,19 @@ search_for = Suche nach '' {0} ''
...
+advanced_search_tip_four = Wortvariationen werden auch gefunden - Bsp.: <i>Sequenz</i> findet <i>Sequenzen</i> and <i>Sequenzierung</i>.
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
advanced_search_tip_four = Wortvariationen werden auch gefunden - Bsp.: <i>Sequenz</i> findet <i>Sequenzen</i> und <i>Sequenzierung</i>.
```

**Review by bkampe — CHANGES_REQUESTED** [review]
Minor spelling correction in german translation

**Review by bkampe — APPROVED** [review]

**Pull request merged**